dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task230420AlterVarcharLengthOfLockedByCol.java

```diff
@@ -0,0 +1,39 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.startup.StartupTask;
+
+import java.sql.SQLException;
+
+/**
+ * This class alter the locked_by column in four different tables.
+ * The locked_by column in some coses has a length of 36. With this class
+ * we are increasing that length to 100.
+ */
+public class Task230420AlterVarcharLengthOfLockedByCol implements StartupTask {
+    private void alterTables() throws SQLException {
+        DotConnect dc = new DotConnect();
+        dc.executeStatement("alter table contentlet_version_info alter column locked_by type varchar (100);");
```

Comment:
Let's remove the semicolon `;` at the end of each of the queries, we have found that something it complains about it and makes the UT fail.

Reply:
All `;` removed
<img width="901" alt="image" src="https://user-images.githubusercontent.com/127987858/234043865-ffc244d9-7a04-4214-a86a-c94a3de7ecce.png">